apps/app/src/lib/datadog.init.tsx

```diff
@@ -32,5 +32,14 @@ export default function DataDogInit() {
     loadDataDogLazily();
   }, []);
 
+  // useSubscribeToUserAuthEvents({
+  //   onSignIn: (user) => {
+  //     datadogLogs.setUser({ id: user.id });
+  //   },
+  //   onSignOut: () => {
+  //     datadogLogs.clearUser();
+  //   },
+  // });
+
```

Comment:
à garder ?

Reply:
Ah non, à supprimer of course, merci, désolé j'ai pas bien vérifié.